dev/resources/schema.setup.cql

```diff
@@ -5,11 +5,10 @@ CREATE TABLE IF NOT EXISTS lcmap_landsat_dev.sources (
     id text,
```

Comment:
Yes, we add a row to a partition for each progress update. This gives us a way to determine what happened any time in the future.